extensions/objects/templates/ListOfSetsOfHtmlStringsEditor.js

```diff
@@ -72,15 +97,21 @@ oppia.directive('listOfSetsOfHtmlStringsEditor', [
                 } else {
                   $scope.value[selectedRank].push(choiceHtml);
                 }
-
-                if ($scope.value[i] === []) {
-                  // Continuity error.
-                  errorMessage = ('No item(s) is assigned at position ' +
-                    String(i + 1) + '. Please assign some item at this ' +
-                    'position.');
-                }
-                choiceHtmlHasBeenAdded = true;
-                break;
+              }
+              choiceHtmlHasBeenAdded = true;
+              break;
+            }
+          }
+          for (var i = 0; i < $scope.value.length; i++) {
+            if ($scope.value[i].length === 0) {
+              if (i === $scope.value.length - 1) {
+                // If it is empty list at the last, pop it out.
+                $scope.value.pop();
+              } else {
+                // Continuity error.
+                errorMessage = ('No item(s) is assigned at position ' +
```

Comment:
`No choice(s) is assigned ...`. Replace `item` with `choice`.

Reply:
Done